src/sagas/RootSaga.js

```diff
@@ -0,0 +1,39 @@
+import { call, put, takeEvery, delay } from 'redux-saga/effects';
+
+import {
+  SAVE_USER_PROFILE,
+  saveUserProfileBegin,
+  saveUserProfileSuccess,
+  saveUserProfileFailure,
+  saveUserProfileReset,
+} from '../actions/profileActions';
+
+import * as UserProfileApiService from '../services/UserProfileApiService';
+
+function* saveUserProfile(action) {
+  try {
+    yield put(saveUserProfileBegin());
+    const userAccount = yield call(
+      UserProfileApiService.saveUserProfile,
```

Comment:
Do we need to update the `userAccount` redux state with the updated data returned from the PATCH request to the user account API?

Reply:
Yes - that's done a few lines down from here where it manually creates a fetchUserAccountSuccess action.

Reply:
Ideally the data would live in some common place where interested UIs could get to it, but for the foreseeable future - since the fetch user account and save user profile methods both return identical structures, I just let userAccount be the authority on that.

Reply:
Ah, got it. Thanks for pointing it out.